test-network-function/identifiers/identifiers.go

```diff
@@ -323,7 +323,9 @@ the same hacks.'`),
 		Description: formDescription(TestOperatorInstallStatusIdentifier,
 			`Ensures that CNF Operators abide by best practices.  The following is tested:
 1. The Operator CSV reports "Installed" status.
-2. TODO: Describe operator scc check.`),
+2. Control the actions that a pod can perform and what it has the ability to access,
```

Comment:
maybe change from  what it has the ability to access to capabilities it has to access

Reply:
maybe I can to adding : In this test we check whether clusterPermissions are set for specific
ClusterServiceVersion is installed. We test operator which is not
using the permissions but this tests enable us to check whether operator
in test is using privileged and in this case we should fail and
analyze why such permission is needed.

Reply:
Ying and I met about this, we propose the following:
Check that the operator is not configured with privileged rights. If clusterPermissions exists in the CSV, and the clusterPermissions[].rules[].resourceNames is not empty then we fail the test. Otherwise, the test passes. In case the test fails, we need to analyze the need for the privilege mode.